public/app.jsx

```diff
@@ -1,9 +1,59 @@
 import React, {useState} from 'react';
-import {render} from 'react-dom';
+import { render } from 'react-dom';
+import axios from 'axios';
 
 function App() {
-  return (
-    <div>hello</div>
+  const [searchTerm, setSearchTerm] = useState('');
+  const [result, setResult] = useState([]);
+
+  const handleChange = (e) => {
+    setSearchTerm(e.target.value);
+    console.log(searchTerm);
+
+    axios.post(`/search`, {searchTerm: e.target.value })
+      .then(res => {
+        setResult(res.data);
+      }).catch(err => {
+      console.log(err)
+    })
+  }
+
+  function getHighlights(r, searchTerm) {
```

Comment:
So what I meant by highlighting the search result is when you search something in google, it gives you a list of results. Then press up and down arrow on the keyboard to highlight different results. Maybe try that one.

<img width="1135" alt="Screen Shot 2020-11-22 at 11 18 30 PM" src="https://user-images.githubusercontent.com/10586163/99928875-0ec29e80-2d19-11eb-8ba5-8fb7f6735dab.png">

Reply:
Notice how the result is highlighted in grey and when I press Enter, it will do a search.
When you press Enter here, you can show a message saying:
"You selected: <search_result>"

Where search_result is the currently highlighted search result.